cmd/res/configuration.toml

```diff
@@ -38,11 +38,12 @@ Type = 'consul'
   Port = 48081
 
 [MessageQueue]
-Enabled = false
-Protocol = 'tcp'
+Protocol = 'redis'
```

Comment:
miss `UseMessageBus = false` under `Service` section

Reply:
Humm, I remember double checking that... :-( Good catch!